autoload/lsp/completion.vim

```diff
@@ -263,7 +263,7 @@ export def CompletionReply(lspserver: dict<any>, cItems: any)
       d.icase = 1
     endif
 
-    if item->has_key('kind')
+    if item->has_key('kind') && !item.kind->empty()
```

Comment:
It is `null` and type is `special`. You know better vim9 script. Let me know how to compare and I will incorporate. Or do it from your end.
When I printed out the `item`, this is what I got: `{'label': '''H''', 'data': {'doc_uri': 'file:///filename.py'},'sortText': 'a''H''', 'kind': null, 'insertText': '''H'''}`.

Reply:
Can you try using `item.kind != null` with pylsp?

Reply:
aha, interesting, @yegappan how that 'kind' became a 'null' ?  vim9 script bug? kind should be a number.

Reply:
yes, it is special value in json, but when it decode to vim9 var, how that handled?
i may think it should be 0 here since kind is number, but not sure where if it checked that.
or anyway as i said pylsp may should not pass a null to kind to vim9 here, it should be its bug also.

Reply:
When decoding a JSON string, Vim doesn't know that a null value is a number and not a string.  There is no type information in JSON.

Reply:
ok, it is a weird result anyway, or even it was convert to 0, should be incorrect also, became unknown situation.
though this PR avoid one place of 'kind' search, it maybe still check/get that somewhere later.
so, better solution i supposed lsp server here should not pass 'null', it is wrong.
ping @ccordoba12